implementing /contributions/count endpoint (#121)
GIScience/ohsome-api: docs/endpoints.rst

      }

Contributions Aggregation
-------------------------
      
.. http:post :: /contributions/count

   Get the count of the contributions provided to the OSM data. This endpoint does not support the deprecated ``types``, ``keys``, ``values`` parameters.

**Example request**:

Number of contributions to the building 'Stadthalle Heidelberg' between 2010 and 2020.

  .. tabs::

    .. code-tab:: bash curl (GET)

>        curl -X GET 'https://api.ohsome.org/v1/contributions/count?bboxes=8.699053,49.411842,8.701311,49.412893&filter=id:way/140112810&time=2010-01-01,2020-01-01

Reviewer: The forgotten single quote at the end of this URL causes a warning at line 1037 creating the docs with Sphinx

Author: Good catch.. my sphinx did not give that specific warning.. just something cryptic about line 4 in this file, but after that fix the warning went away 🤷

	  },
	  "apiVersion":"1.4.0",
	  "result":[
	    {
	      "fromTimestamp":"2010-01-01T00:00:00Z",
	      "toTimestamp":"2020-01-01T00:00:00Z",
	      "value":15.0
	    }
	  ]
	}

   .. code-tab:: json curl (POST)

	{
	  "attribution":{
	    "url":"https://ohsome.org/copyrights",
	    "text":"© OpenStreetMap contributors"
	  },
	  "apiVersion":"1.4.0",
	  "result":[
	    {
	      "fromTimestamp":"2010-01-01T00:00:00Z",
	      "toTimestamp":"2020-01-01T00:00:00Z",
	      "value":15.0
	    }
	  ]
	}

   .. code-tab:: json Python

	{
	  "attribution":{
	    "url":"https://ohsome.org/copyrights",
	    "text":"© OpenStreetMap contributors"
	  },
	  "apiVersion":"1.4.0",
	  "result":[
	    {
	      "fromTimestamp":"2010-01-01T00:00:00Z",
	      "toTimestamp":"2020-01-01T00:00:00Z",
	      "value":15.0
	    }
	  ]
	}

   .. code-tab:: json R

	{
	  "attribution":{
	    "url":"https://ohsome.org/copyrights",
	    "text":"© OpenStreetMap contributors"
	  },
	  "apiVersion":"1.4.0",
	  "result":[
	    {
	      "fromTimestamp":"2010-01-01T00:00:00Z",
	      "toTimestamp":"2020-01-01T00:00:00Z",
	      "value":15.0
	    }
	  ]
	}

.. http:post :: /contributions/count/density

>    Get the density of the count of contributions divided by the total area in square-kilometers. This endpoint does not support the deprecated ``types``, ``keys``, ``values`` parameters.

Reviewer 2: ```suggestion
   Get the density of the count of contributions in the total query area in counts per square-kilometers. This endpoint does not support the deprecated ``types``, ``keys``, ``values`` parameters.
```

Author: This wording would then not fit to the wording used in the other density description in line 134.

Reviewer 2: but this wording is the correct one :stuck_out_tongue: becuase a density of something divided by the area would have the unit of `thing / length^4` (instead of `thing / lenght^2`)

Author: Yeah I get now that you can also understand the old wording in a different way.. Then just fix also the old wording in the other line and it's fine :)

   Get the count of the contributions provided to the OSM data. This endpoint does not support the deprecated ``types``, ``keys``, ``values`` parameters.

> .. note:: The /contributions/count endpoint is a new feature that is in the experimental status, meaning it is still under internal evaluation and might be subject to changes in the upcoming minor or patch releases.

Reviewer: Nice!
